templates/projects/empty-server/files/server/middleware.json

```diff
@@ -47,6 +47,5 @@
     "loopback#urlNotFound": {}
   },
   "final:after": {
-    "loopback#errorHandler": {}
```

Comment:
That was used initially, then we agreed to use `errorhandler` instead. .. reverting back now. Fixed. 
Also, same for `middleware.development.json` .. file was there then was removed since we didn't need the errorhandler middleware anymore. I reverted and fixed the issue. 
All in place now.